src/System.Net.Requests/src/System.Net.Requests.csproj

```diff
@@ -52,6 +52,10 @@
     <Compile Include="$(CommonPath)\System\NotImplemented.cs">
       <Link>Common\System\NotImplemented.cs</Link>
     </Compile>
+    <ProjectReference Include="..\..\System.Net.Http\pkg\System.Net.Http.pkgproj">
```

Comment:
Implementations don't consume implementations, they compile against references.  The implementation used at run-time depends on what gets put in the shared framework / what gets referenced by the app.

Moreover: typically we don't couple fixes between libraries in servicing.  We keep them independent so that folks don't have to update multiple components together.